lib/beacon/loader/module_loader.ex

```diff
@@ -1,20 +1,29 @@
 defmodule Beacon.Loader.ModuleLoader do
-  def load(module, code_string) do
+  def load(module, ast) do
     :code.delete(module)
```

Comment:
I don't think you need (nor should use) either `:code.delete/1` or `:code.purge/1` here. If there are ongoing requests to the server, this will result in those requests failing badly if they happen to be requesting something which references the affected code. You should leave this behavior to the ERTS code loader itself, which will ensure that ongoing requests complete using the previous version of the code, and any new requests use only the new code.

Reply:
this would only get kicked off once on a page CRUD event. But with the move towards storing compiled pages in ets the need for this at all should go away